includes/application/forwardApprover.php

```diff
@@ -63,20 +63,38 @@
 	if( isset( $_SESSION['PostName'] ) )
 	{
 		$UID = $_SESSION['PostName'];
+		if ( forwardApprover( $app_id ,$app_type, $UID ) == true )
+		{
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->